docs/web-apps.qmd

```diff
@@ -11,23 +11,25 @@ This is a great way to quickly test your model, but you'll likely want to embed
 
 ## Shiny
 
-Pass the result of `.chat()` directly to Shiny's [`ui.Chat` component](https://shiny.posit.co/py/components/display-messages/chat/) to create a chat interface in your own [Shiny](https://shiny.rstudio.com/py) app.
+Pass the result of `chat.stream()` directly to Shiny's [`ui.Chat` component](https://shiny.posit.co/py/components/display-messages/chat/) to create a chat interface in your own [Shiny](https://shiny.rstudio.com/py) app.
```

Comment:
On second read, I don't think I got this right and I'm not quite sure what you were wanting to say here; I think maybe the example is out of sync with the text.

Reply:
Ah, I mean to say to pass it to `.append_message_stream()`